arviz/data/io_netcdf.py

```diff
@@ -39,3 +39,7 @@ def save_data(data, filename, *, group="posterior", coords=None, dims=None):
     """
     inference_data = convert_to_inference_data(data, group=group, coords=coords, dims=dims)
     return inference_data.to_netcdf(filename)
+
+
+from_netcdf = load_data  # pylint: disable=invalid-name
```

Comment:
Can we just rename the functions rather than alias them and have two of the same function in the API?

Reply:
We could. Then we only need to mention this change in "what's changed". Or deprecate old functionality.

Reply:
I vote for deprecate old functionality. If we want to be safe we can wrap load_data and add a warning

Reply:
i'm fine with deprecation, or using the fact that we're not 1.0 yet to just change it (starting up a RELEASE_NOTES.md is a good idea!)